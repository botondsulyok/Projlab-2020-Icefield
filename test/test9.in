field stable 0
field unstable 1 false 0
neighbours stable unstable
neighbours unstable stable
entity mover Eskimo stable
stats stable
stats unstable
stats mover
move mover unstable
stats mover
stats unstable