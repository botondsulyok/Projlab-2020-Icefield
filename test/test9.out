id: stable
type: iceunit
building: plains0
snow: 0
pickable: null
standers: mover
pullers: mover
masters: mover
neighbours: unstable

id: unstable
type: unstable
building: plains1
snow: 0
pickable: null
standers: null
pullers: null
masters: null
neighbours: stable
capacity: 1
broken: false

id: mover
type: eskimo
iceUnit: stable
movebehaviour: walk0
bodyheat: 5
stamina: 4
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: null
components: null

Game Over
id: mover
type: eskimo
iceUnit: unstable
movebehaviour: walk0
bodyheat: 5
stamina: 3
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: null
components: null

id: unstable
type: unstable
building: plains2
snow: 0
pickable: null
standers: mover
pullers: mover
masters: mover
neighbours: stable
capacity: 1
broken: true

